.github/workflows/test-windows.yml

```diff
@@ -39,6 +39,12 @@ jobs:
         run: make build-cli-windows-amd
         shell: pwsh
 
+      # Builds an init package manually off of the last release
+      # Windows in GitHub cannot natively build linux containers
+      - name: Build init-package
+        run: |
+          make release-init-package ARCH=amd64 AGENT_IMAGE=agent:v0.23.6
```

Comment:
since we aren't actually using this for these tests, can we leave a note stating that, maybe we can even make a custom tag that just says "ignore-me" or something for this?

Reply:
I'll add a note (someone poking in our package registry may get confused if they find the tag lol)